uwblueprint/community-fridge-kw
# feat: Create a backend service to call all checkins and food rescues associated with a volunteer

## Brief description. What is this change? 

### [Create a backend service to call all checkins and food rescues associated with a volunteer](https://www.notion.so/uwblueprintexecs/Create-a-backend-service-to-call-all-checkins-and-food-rescues-associated-with-a-volunteer-96bf987e87e84e51b68cd6cb8b7eec05)


## Implementation description. How did you make this change?
* Created a GET endpoint in the volunteer service that calls both checkins and schedulings and sorts entries by last updated (not sure if the sorted times is what we want?)
* Cleaned up some of the backend tests that were using redundant code



## Steps to test
1. Seed up the database or create a volunteer with both checkins and schedulings. Run the endpooint at `localhost:5000/volunteers/shifts/<volunteerId>`
2. Ensure that is everything is returned in from most recent to least and that a type of 'scheduling' or 'checkin' is attached to each of them.

## Checklist
